### PR TITLE
RUC-19 Feature breadcrumbs component storybook

### PR DESCRIPTION
**Pull request by monikasudol**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link href="https://fonts.googleapis.com/css2?family=Amiri&display=swap" rel="stylesheet">
```
Comment (pdobrowolski99) [inline public/index.html:13]: Why not Montserrat like you can see inside designs? Amiri has special style and font weight 400, 700 only

File: src/App.scss [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.active {
+  font-weight: bold;
+  font-family: 'Amiri', serif;
+}
+
+.breadcrumbsWrapper {
```
Comment (pdobrowolski99) [inline src/App.scss:15]: It's better to create stylesheet for single component, not general stylesheet

File: public/index.html (hunk 1 above)
Comment (monikasudol) [reply public/index.html:13]: Sure, it's just for style used by user, if he want to use own (so just used in example). I've left that for this example, but also added Monterserrat and set it on body to have it default (in index.scss)

File: src/App.scss (hunk 2 above)
Comment (monikasudol) [reply src/App.scss]: Improved, it's just class used in example, so I've moved it and added comment to remove it later (when remove example) :)

**Pull request merged**